### PR TITLE
Translate to English

### PR DESCRIPTION
**Pull request by Pierstoval**
Wasn't really straightforward, I tried to follow the docs, and it seems that it works on my local machine, so 😄 

Next step: we need to translate the base website!

### CODE REVIEW

File: src/translations/en.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  body: '
+      Against the destruction of our social model.<br />
+      Against the constant growth of inequality.<br />
+      Against the systematic stigmatization of minorities.<br />
+      <span class="dcs-website-name__default">This</span>
+      <span class="dcs-website-name__prefix">The</span>
+      <span class="dcs-website-name__website">website</span>
+      <span class="dcs-website-name"></span>
+      is on strike.'
+  ready:  'Read the call ➔'
```
Comment (noelmace) [inline src/translations/en.yml]: I'm not sure "call" is correct here.
What do you think about "our manifesto" instead?

File: src/translations/en.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  footer: 'You will be able to access this website on <time id="dcs-tomorrow-date" datetime=""></time>'
+footer:
+  body: 'Our website is against the pensions reform and will be on strike on <time id="dcs-strike-date" datetime=""></time>. And you?'
```
Comment (noelmace) [inline src/translations/en.yml]:
```suggestion
  body: 'Our website is against the french pensions reform and will be on strike on <time id="dcs-strike-date" datetime=""></time>. And you?'
```

File: src/translations/en.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  ready: 'I&apos;m in. ➔'
+
+title: 'Digital strike against the pensions reform and its camp'
```
Comment (noelmace) [inline src/translations/en.yml]:
```suggestion
title: 'Digital strike against the french pensions reform and its camp'
```

**Comment by noelmace** [comment]
@dunglas should we old this PR until we have translated the website?

File: src/translations/en.yml (hunk 2 above)
Comment (dunglas) [reply src/translations/en.yml]: French

**Comment by dunglas** [comment]
> @dunglas should we old this PR until we have translated the website?

I don't think so, we should merge ASAP.

File: src/translations/en.yml (hunk 1 above)
Comment (Pierstoval) [reply src/translations/en.yml]: Go for manifesto, that's better :)

**Comment by Pierstoval** [comment]
Fixed reviews :)

**Pull request merged**